<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAinDataMapDlg</class>
 <widget class="QDialog" name="QAinDataMapDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,3">
     <item>
      <layout class="QVBoxLayout" name="m_LeftLayout">
       <item>
        <widget class="QLabel" name="m_labTip">
         <property name="text">
          <string>在左侧表格选中需要映射的参数后，在右侧表格双击需要映射的模型数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="m_RightLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="m_labDataSet">
           <property name="text">
            <string>数据集：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_cmbDataSet"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="m_btnAdd">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnDelete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnClearAllMap">
       <property name="text">
        <string>清空所有映射</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnClearClickedMap">
       <property name="text">
        <string>清除选中映射</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="m_stcDevMeas">
       <property name="text">
        <string>装置采样：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_cmbDevMeas"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_stcPt">
       <property name="text">
        <string>PT变比：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_edtPtPrimary"/>
     </item>
     <item>
      <widget class="QLabel" name="m_stcPtPirmary">
       <property name="text">
        <string>kV /</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_edtPtSecondary"/>
     </item>
     <item>
      <widget class="QLabel" name="m_stcPtSecondary">
       <property name="text">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_stcCt">
       <property name="text">
        <string>CT变比：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_edtCtPrimary"/>
     </item>
     <item>
      <widget class="QLabel" name="m_stcCtPirmary">
       <property name="text">
        <string>A /</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_edtCtSecondary"/>
     </item>
     <item>
      <widget class="QLabel" name="m_stcCtSecondary">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnOK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
